api/apps/geoprocessing/src/migrations/geoprocessing/1646308580694-AddProjectsPuTable.ts

```diff
@@ -0,0 +1,21 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class AddProjectsPuTable1646308580694 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    queryRunner.query(`
+      CREATE TABLE projects_pu(
+        id uuid PRIMARY KEY DEFAULT gen_random_uuid(),
+        project_id uuid NOT NULL,
```

Comment:
Since we dropped `puid` column of `scenarios_pu_data` `(puid, scenario_id)` index has been removed. We can create both, one for `project_id` in `projects_pu` and the other for `scenario_id` in `scenarios_pu_data`. What do you think? 🤔